### PR TITLE
[basic_kbdlvst] update font

### PR DESCRIPTION
**Pull request by DavidLRowe**

### CODE REVIEW

**Comment by DavidLRowe** [comment]
Fixes #1877 (hopefully)

File: release/basic/basic_kbdlvst/source/basic_kbdlvst.kps [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+    <FollowKeyboardVersion/>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
```
Comment (mcdurdin) [inline release/basic/basic_kbdlvst/source/basic_kbdlvst.kps:18]: You'll need to add the dejavu font to the package and reference it in the `<Keyboards>` section also.

**Comment by DavidLRowe** [comment]
@mcdurdin Hopefully these changes are what you had in mind!

**Review by mcdurdin — APPROVED** [review]
Yep, those changes look good. LGTM

**Pull request merged**